synapse/storage/data_stores/main/end_to_end_keys.py

```diff
@@ -248,6 +248,73 @@ def _count_e2e_one_time_keys(txn):
 
         return self.runInteraction("count_e2e_one_time_keys", _count_e2e_one_time_keys)
 
+    def _get_e2e_cross_signing_key_txn(self, txn, user_id, key_type, from_user_id=None):
```

Comment:
This is just moving these functions from `EndToEndKeyStore` to `EndToEndKeyWorkerStore` so that it works with workers.